tools/slicec-cs/src/slicec_ext/member_ext.rs

```diff
@@ -40,14 +42,30 @@ impl FieldExt for Field {
             .into_iter()
             .any(|a| a.downcast::<CsReadonly>().is_some())
     }
+
+    fn formatted_param_doc_comment(&self) -> Option<String> {
+        if let Entities::Enumerator(enumerator) = self.parent().concrete_entity() {
+            // Check if the enumerator has a doc comment on it.
+            enumerator.comment().and_then(|comment| {
+                // If it does, search the comment for a '@param' tag with this field's identifier and return it.
+                comment
+                    .params
+                    .iter()
+                    .find(|param_tag| param_tag.identifier.value == self.identifier())
+                    .map(|param_tag| format_comment_message(&param_tag.message, &self.namespace()))
+            })
+        } else {
+            None
```

Comment:
I'd just panic here. If we call this function on anything other than an `Enumerator`, we're doing something wrong. Panicing will alert us to this fact, whereas returning `None` gives the impression this is fine to call on non-enumerator fields.
```suggestion
            panic!("Called 'formatted_param_doc_comment' on field outside of enumerator!");
```